BUILDING_ON_WINDOWS.md

```diff
@@ -28,15 +29,25 @@ Notes:
 #### When prompted which components to install:
 
 1. Unfold the tree element that says "Qt"
-2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 5.15.2`)
+2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 6.5.3`)
```

Comment:
I know this is a bit late, but Qt 6.6.0 will be out tomorrow (most likely), and we noticed a bug in Qt 6.5.3 [not in 6.6], so I think it would be better to use 6.6.0 here, but it's not that big of a deal.